python/tank/authentication/sso_saml2/core/sso_saml2_core.py

```diff
@@ -378,13 +370,13 @@ def update_session_from_browser(self):
 
         cookie_jar = self._view.page().networkAccessManager().cookieJar()
 
-        # Here, the cookie jar is a dictionary of key/values
-        cookies = SimpleCookie()
-
+        # Serializing the cookies in a format compatible with SimpleCookie
```

Comment:
That big PR description you gave was awesome, but it would be great is it was summarized here in the code so someone reading the code knows why you are doing something funny here. If I was looking at the code I'd be asking myself why aren't you just using SimpleCookie in the first place.

Reply:
Updated the comment as suggested.